documentatie/use-cases/GSB-gebruikersdoelen.md

```diff
@@ -107,40 +107,46 @@ __uitbreidingen__:
 - OSV richt de server ook gelijk als client in. Willen wij dit ook?
 - Afzenderverificatie ontbreekt nog, want nog geen beslissing over oplossing.
 
-## GSB richt de applicatie in
+
+## De coördinator maakt de stembureaus aan.
 
 __niveau:__ gebruikersdoel, zee, 🌊
 
-### Hoofdscenario en uitbreidingen
+__precondities:__
 
-__trigger:__
+- De coördinator heeft de lijst met gepubliceerde stembureaus.
+- De invoer is nog niet gestart.
 
-- [GSB installeert de applicatie.](#gsb-installeert-de-applicatie)
-- Het GSB maakt de kandidatenlijst beschikbaar.
-- De Kiesraad maakt de verkiezingsdefinitie beschikbaar.
+### Hoofdscenario en uitbreidingen
 
 __hoofdscenario__:
 
-1. De beheerder leest de verkiezingsdefinitie in.
-2. De beheerder leest de kandidatenlijst in.
-3. De beheerder maakt de stembureaus aan.
-4. De beheerder maakt de gebruikers aan.
+1. De coördinator selecteert de verkiezing.
+2. De coördinator importeert het EML-NL bestand met de stembureaus.
+3. De applicatie stelt vast dat de stembureaus aan de validaties voldoen.
+4. De coördinator stelt vast dat de stembureaus in de applicatie volledig en correct zijn, door ze te vergelijken met de door de gemeente vooraf gepubliceerde lijst.
```

Comment:
Is die vooraf gepubliceerde lijst noodzakelijk om te vermelden? In een kleine gemeente doe je dit waarschijnlijk uit je hoofd?

Reply:
wijziging doorgevoerd